src/index.ts

```diff
@@ -36,7 +36,7 @@ const program = new Command();
 program
   .name('Hexabot')
   .description('A CLI to manage your Hexabot chatbot instance')
-  .version('2.0.0');
+  .version('2.0.2');
```

Comment:
Let's take this value from package.json

Reply:
Hi @marrouchi, feedback applied :heavy_check_mark: